uber/config.py

```diff
@@ -538,3 +538,20 @@ def _is_intstr(s):
     c.VOLUNTEER_CHECKLIST = [url for step, url in _items]
 
 stripe.api_key = c.STRIPE_SECRET_KEY
+
+c.AWESOME_WORDS = [
+    "Fire",
+    "Water",
+    "Air",
+    "Earth",
+    "Satan"
```

Comment:
One of the basic elements! 0:) 
Feel _free_ to **make the change yourself >:)**